source/Ship.cpp

```diff
@@ -4177,7 +4177,11 @@ void Ship::DoGeneration()
 		if(currentSystem)
 		{
 			double scale = .2 + 1.8 / (.001 * position.Length() + 1);
-			fuel += currentSystem->RamscoopFuel(attributes.Get("ramscoop"), scale);
+			fuel += currentSystem->RamscoopFuel(attributes.Get("ramscoop"), 1., scale);
+			fuel += currentSystem->RamscoopFuel(attributes.Get("velocity ramscoop"),
+				velocity.Length() * !IsHyperspacing() / 1.66, scale);
```

Comment:
```suggestion
				IsHyperspacing() ? 0 : velocity.Length() / 1.66, scale);
```
Should be more readable.